staging/src/k8s.io/client-go/tools/remotecommand/websocket_test.go

```diff
@@ -814,6 +815,42 @@ func TestWebSocketClient_BadHandshake(t *testing.T) {
 	}
 }
 
+// See (https://github.com/kubernetes/kubernetes/issues/126134).
```

Comment:
nit: move this to the end to make cherry-picking easier

Reply:
I'll be honest--I don't fully understand this. Should the comment go at the end of the test--or at the end of the file?

Reply:
IIUIC is to move ALL the new code added at the end of the file

Reply:
Done